python/dgl/sparse/sparse_matrix.py

```diff
@@ -680,6 +680,74 @@ def sample(
             self.c_sparse_matrix.sample(dim, fanout, ids, replace, bias)
         )
 
+    def relabel(
```

Comment:
compact is probably better name, relabel usually infer there is a input map from A to B.

Reply:
@czkkkkkk @peizhou001 WDYT?

Reply:
Make sense!